### ISSUE TITLE
[SUGGESTION] Implement a Antimatter Bomb Pointer for the Covenant to use in the event of bomb loss.

### ISSUE THREAD

**Issue by SharkLordSatan** [HaloSpaceStation/HaloSpaceStation13#3102]
#### Description of issue

Currently, there's a (temporary) rule on the Discord that states that Covenant are allowed to use Slipspace Core Bombs from summoned ships if they cannot locate the Antimatter Bomb for whatever reason. However, retrieval of said Antimatter Bomb, should it fall into UNSC hands, can be incredibly difficult under most circumstances, as the majority of the time they'll place it in hard-to-find places and ensure that access to said bomb is difficult by bolt-locking doors or whatever. Which means if the Covenant opts to try and find the bomb, they'll more likely than not end up wasting precious time and manpower to find and secure the bomb.

So, I propose this: an Antimatter Bomb Pointer, which would function basically identically to the Scanning Point and Artifact Pointers, except that it points in the direction of the Antimatter Bomb. To make it Fair and Balanced™ the entire CRS cruiser probably only gets one, located somewhere on the Bridge (perhaps in the small room that has the AI chip storage).

Basically it means that the Covenant has a way of actually HUNTING DOWN the bomb instead of wasting time searching every possible spot on the ODP while simultaneously fighting ODSTs, marines, the ODP AI, and of course, locked doors. Should the Covenant fail to find the bomb and/or lose the Antimatter Bomb Pointer, it'll then encourage them (should they decide to do so) to call in a support ship and use that ships' Slipspace Core Bomb.

**Comment by Aroliacue** [comment]
Personally I think that Slipspace Core Bombs should be immediately allowed if the first bomb falls into enemy hands. It still requires some amount of logistics but due to the current state of ODP phase, it's a much better choice to ensure the game progresses at a decent pace and doesn't immediately become a UNSC Victory. (To elaborate, losing the bomb is basically a guaranteed loss for the Covenant, unless the UNSC puts it in a bad spot/doesn't defend it properly, as was the case when it was placed in the UNSC bunks)

**Comment by X0-11** [comment]
A different method of handling this has been taken.
